core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/AggregatedWorkspaceService.scala

```diff
@@ -59,10 +59,9 @@ class AggregatedWorkspaceService(workspaceManagerDAO: WorkspaceManagerDAO) exten
             )
           case WorkspaceType.McWorkspace =>
             val id = workspace.workspaceIdAsUUID
-            wsmResponse
-              .find(_.getId == id)
+            wsmResponse.get(id)
               .map(wsmInfo =>
-                Try(aggregateMCWorkspaceWithWSMInfo(workspace, wsmInfo)).recover {
+                Try(aggregateMCWorkspaceWithWSMInfo(workspace, wsmInfo.head)).recover {
```

Comment:
Why `wsmInfo.head` here?

Reply:
`groupBy` gives the type `Map[Key, List[Value]]` (scala doesn't provide a built-in to convert to `Map[Key, Value]`).
`wsmInfo.head` just takes the first item from that list (which will be of exactly 1 element).